schema/artwork/index.js

```diff
@@ -435,20 +435,23 @@ const ArtworkType = new GraphQLObjectType({
       },
       sale_artwork: {
         type: SaleArtwork.type,
-        resolve: ({ id }) =>
-          gravity('related/sales', { artwork: [id], active: true, size: 1 })
```

Comment:
`active: true` does nothing for the related endpoint, so it looks like this was just requesting _any_ related sale, and then requesting the artwork's sale artwork in that sale.